src/HonzaBotner.Discord/DiscordBot.cs

```diff
@@ -18,17 +21,19 @@ internal class DiscordBot : IDiscordBot
         private readonly EventHandler.EventHandler _eventHandler;
         private readonly CommandConfigurator _configurator;
         private readonly IVoiceManager _voiceManager;
+        private readonly IOptions<DiscordConfig> _discordOptions;
 
         private DiscordClient Client => _discordWrapper.Client;
         private CommandsNextExtension Commands => _discordWrapper.Commands;
 
         public DiscordBot(DiscordWrapper discordWrapper, EventHandler.EventHandler eventHandler,
-            CommandConfigurator configurator, IVoiceManager voiceManager)
+            CommandConfigurator configurator, IVoiceManager voiceManager, IOptions<DiscordConfig> discordOptions)
         {
             _discordWrapper = discordWrapper;
             _eventHandler = eventHandler;
             _configurator = configurator;
             _voiceManager = voiceManager;
+            _discordOptions = discordOptions;
```

Comment:
The property should be `DiscordConfig`, not `IOptions<DiscordConfig>`, e.g. in the constructor it should be ` _discordOptions = discordOptions,Value;`.